--- neformal-1.0.1-orig/mainwindow.cpp	2009-04-16 23:46:14.000000000 +0200
+++ neformal-1.0.1/mainwindow.cpp	2009-04-17 22:43:25.072348971 +0200
@@ -75,11 +75,11 @@
   text_file_browser = new QPlainTextEdit;
   text_file_browser->setReadOnly (true);
   
-  qApp->setWindowIcon (QIcon (":/icons/app.svg"));
+  qApp->setWindowIcon (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/app.svg"));
  
   if (QSystemTrayIcon::isSystemTrayAvailable())
      {
-      ico_tray = new QSystemTrayIcon (QIcon (":/icons/app.svg"), this);
+      ico_tray = new QSystemTrayIcon (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/app.svg"), this);
       ico_tray->show(); 
       connect (ico_tray, SIGNAL(activated ( QSystemTrayIcon::ActivationReason)), 
               this, SLOT(tray_activated ( QSystemTrayIcon::ActivationReason)));
@@ -387,7 +387,7 @@
 
 void MainWindow::sourceChanged (const Phonon::MediaSource &source)
 {
-  act_play->setIcon (QIcon (":/icons/play.svg"));
+  act_play->setIcon (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/play.svg"));
     
   QModelIndex index = fman->index_from_name (QFileInfo(mediaObject->currentSource().fileName()).fileName());
 
@@ -483,20 +483,20 @@
 
 void MainWindow::create_actions()
 {
-  act_test = new QAction (QIcon (":/icons/play.svg"), tr ("Test"), this);
+  act_test = new QAction (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/play.svg"), tr ("Test"), this);
   connect (act_test, SIGNAL(triggered()), this, SLOT(test()));
   
   
-  act_play = new QAction (QIcon (":/icons/play.svg"), tr ("Play"), this);
+  act_play = new QAction (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/play.svg"), tr ("Play"), this);
   connect (act_play, SIGNAL(triggered()), this, SLOT(play()));
 
-  act_stop = new QAction (QIcon (":/icons/stop.svg"), tr ("Stop"), this);
+  act_stop = new QAction (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/stop.svg"), tr ("Stop"), this);
   connect (act_stop, SIGNAL(triggered()), this, SLOT(stop()));
 
-  act_prev = new QAction (QIcon (":/icons/prev.svg"), tr ("Prev"), this);
+  act_prev = new QAction (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/prev.svg"), tr ("Prev"), this);
   connect (act_prev, SIGNAL(triggered()), this, SLOT(hard_prev()));
   
-  act_next = new QAction (QIcon (":/icons/next.svg"), tr ("Next"), this);
+  act_next = new QAction (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/next.svg"), tr ("Next"), this);
   connect (act_next, SIGNAL(triggered()), this, SLOT(hard_next()));
   
   act_exit = new QAction (tr ("Exit"), this);
@@ -774,13 +774,13 @@
   if (mediaObject->state() == Phonon::PausedState) 
      {
       mediaObject->play();
-      act_play->setIcon (QIcon (":/icons/play.svg"));
+      act_play->setIcon (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/play.svg"));
      }
   else
   if (mediaObject->state() == Phonon::PlayingState) 
     {
      mediaObject->pause();
-     act_play->setIcon (QIcon (":/icons/pause.svg"));
+     act_play->setIcon (QIcon ("/usr/share/icons/hicolor/scalable/apps/neformal/pause.svg"));
     }
   else
   if (mediaObject->state() == Phonon::StoppedState || Phonon::ErrorState) 
